src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -87,7 +87,7 @@ public function overviewAction()
      *
      * Parse all errors mapped by id html field
      *
-     * @param Form $form The form
+     * @param Form $form
```

Comment:
Reminded me of
<img src="https://pbs.twimg.com/media/D-YGN_EXsAAu6nl?format=png&name=small">